plugins/SitesManager/SitesManager.php

```diff
@@ -514,6 +514,12 @@ public function getClientSideTranslationKeys(&$translationKeys)
         $translationKeys[] = "SitesManager_SiteWithoutDataGoogleTagManagerDescription";
         $translationKeys[] = "SitesManager_SiteWithoutDataWordpress";
         $translationKeys[] = "SitesManager_SiteWithoutDataWordpressDescription";
+        $translationKeys[] = "SitesManager_SiteWithoutDataStartTrackingDataHeader";
+        $translationKeys[] = "SitesManager_SiteWithoutDataStartTrackingDataDescriptionLine1";
+        $translationKeys[] = "SitesManager_SiteWithoutDataStartTrackingDataDescriptionLine2";
+        $translationKeys[] = "SitesManager_EmailInstructionsButtonText";
+        $translationKeys[] = "SitesManager_SiteWithoutDataIgnorePage";
```

Comment:
This one seems to "replace" `SiteWithoutDataIgnoreMessage`, so guess the old one can be removed.